Tiltfile

```diff
@@ -124,14 +181,19 @@ def fixup_yaml_empty_arrays(yaml_str):
     return yaml_str.replace("storedVersions: null", "storedVersions: []")
 
 ## This should have the same versions as the Dockerfile
-tilt_dockerfile_header_cspo = """
-FROM docker.io/alpine/helm:3.12.2 as helm
```

Comment:
Thanks for removing Helm, but I suggest removing Helm from the whole repository(e.g. Makefile, README.md, .gitignore, .github/)

Reply:
Please remove from .gitignore also